tests/test_utils.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+rng = np.random.default_rng()
+
+from iterative_ensemble_smoother.utils import _compute_AA_projection
+
+
+def test_that_svd_projection_is_same_as_pinv():
+    N = 10
+    p = 5
+    X = rng.standard_normal(size=(p, N))
+    A = X - X.mean(axis=1, keepdims=True)
```

Comment:
Use words or comment what these symbols mean.

Reply:
Do have suggesting for good words or comments to use to describe a matrix of numbers?

Reply:
Maybe at least replace `N` with `realizations` and `p` with `parameters`.

Reply:
Done